### ISSUE TITLE
Include src/ folder

### ISSUE THREAD

**Issue by Mitchellbrine** [NOVA-Team/NOVA-Template#3]
If you run gradle without a src/main folder, it will not add it to the project (at least I've come across that issue before).  
It's a simple fix, I know, but it's worth noting that it should exist. (It can be blank).

**Comment by RX14** [comment]
Yeah I know, I need to do some shit with NovaGradle to get this done.

**Comment by Mitchellbrine** [comment]
Okie
